AC_CHECK_FILE is not available for crosscompiling
error: cannot check for file existence when cross compiling

http://cgit.freedesktop.org/xorg/xserver/commit/?id=56658fc06cb773ed20ef1b23a3fe918c0d3b36bb
===================================================================
--- a/configure.ac.orig	2010-02-10 15:41:33.000000000 +0100
+++ b/configure.ac	2010-02-10 15:41:51.000000000 +0100
@@ -505,14 +505,14 @@
 
 dnl Uses --default-font-path if set, otherwise checks for /etc/X11/fontpath.d,
 dnl otherwise uses standard subdirectories of FONTROOTDIR
-AC_CHECK_FILE([${sysconfdir}/X11/fontpath.d],
-	[DEFAULT_FONT_PATH='catalogue:${sysconfdir}/X11/fontpath.d'],
-	[
-		DEFAULT_FONT_PATH="${FONTMISCDIR}/,${FONTTTFDIR}/,${FONTOTFDIR}/,${FONTTYPE1DIR}/,${FONT100DPIDIR}/,${FONT75DPIDIR}/"
-		case $host_os in
-			darwin*)    DEFAULT_FONT_PATH="${DEFAULT_FONT_PATH},/Library/Fonts,/System/Library/Fonts" ;;
-		esac
-	])
+dnl AC_CHECK_FILE([${sysconfdir}/X11/fontpath.d],
+dnl 	[DEFAULT_FONT_PATH='catalogue:${sysconfdir}/X11/fontpath.d'],
+dnl 	[
+dnl 		DEFAULT_FONT_PATH="${FONTMISCDIR}/,${FONTTTFDIR}/,${FONTOTFDIR}/,${FONTTYPE1DIR}/,${FONT100DPIDIR}/,${FONT75DPIDIR}/"
+dnl 		case $host_os in
+dnl 			darwin*)    DEFAULT_FONT_PATH="${DEFAULT_FONT_PATH},/Library/Fonts,/System/Library/Fonts" ;;
+dnl 		esac
+dnl 	])
 AC_ARG_WITH(default-font-path, AS_HELP_STRING([--with-default-font-path=PATH], [Comma separated list of font dirs]),
 				[ FONTPATH="$withval" ],
 				[ FONTPATH="${DEFAULT_FONT_PATH}" ])
